modules/ztp-pre-staging-booting-from-live-os.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-booting-from-live-os_{context}"]
+= Booting from a live operating system image
+
+You can use the {factory-prestaging-tool} with servers where only one disk is available and external disk drive cannot be attached to the server.
+
+[WARNING]
+====
+{op-system} requires the disk to not be in use when the disk is about to be written by an {op-system} image.
+====
+
+You can mount the {op-system} live ISO on the blank server with one of the following methods:
+
+* Using the Dell RACADM tool on a Dell server.
+* Using the HPONCFG tool on a HP server.
+* Using the Redfish BMC API.
+
+[NOTE]
+====
+It is recommended to automate the mounting procedure. To automate the procedure, you need to pull the required images and host them on a local HTTP server.
+====
+
+.Prerequisites
+
+* You installed `podman`
+* You powered up the host
+* You have network connectivity to the host
+
+.Procedure
+
+This example procedure uses the Redfish BMC API to mount the {op-system} live ISO.
```

Comment:
We need to mention something like: "Verify that your server supports the Redfish interface". There are some that the Redfish implementation is not the standard one and there are other vendors where you need to purchase an extra license for enabling Redfish.